src/yuzu/main.cpp

```diff
@@ -979,6 +979,42 @@ bool GMainWindow::LoadROM(const QString& filename) {
     return true;
 }
 
+QString GMainWindow::LoadDLCsName(u64 title_id) {
+    const FileSys::PatchManager patch_manager(title_id);
+    std::string dlc = "";
```

Comment:
```suggestion
    QString dlc;
```

Convert where needed in the function for localization reasons, which I'll provide below.